### PR TITLE
FIX : Handle checkpoint error and proceed with remaining ops deletion batch

### PR DESCRIPTION
**Pull request by arafat-java**
## Description

Presently when using the deleteSummarizedOps.ts when the aggregated data regarding all the unique documentIds is passed, the process iterates through each of those and then adds soft delete markers to deltas (where applicable) and does hard delete of deltas (where applicable). And this is done in a sequential manner for each of the documents which are passed in.
Now the issue is whenever there is an error occurring for any of the documents, all the documents there after are not handled and the loop breaks in between
Ideally the error should be logged for the document for which the failure happens and all the subsequent documents deletion should take place
This commit intends to handle the checkpoint fetch error for specific documents gracefully, log the details of that document and proceed with processing rest of the batch

### CODE REVIEW

**Comment by arafat-java** [comment]
@tylerbutler @anthony-murphy 
I see you guys have some commits on deleteSummarizedOps.ts
My PR is stuck with Validate CODEOWNERS Expected — Waiting for status to be reported 
Any idea how to proceed on this

**Comment by arafat-java** [comment]
This PR fixes https://github.com/microsoft/FluidFramework/issues/20629

File: server/routerlicious/packages/services-utils/src/deleteSummarizedOps.ts [hunk 1]
```diff
@@ -45,6 +45,14 @@ export async function deleteSummarizedOps(
 				doc.documentId,
 			);
 
+			if (realDoc === null) {
```
Comment (zhangxin511) [inline server/routerlicious/packages/services-utils/src/deleteSummarizedOps.ts:48]: should work for this. But maybe we can do something even safer like:
```
const lastSummarySequenceNumber = JSON.parse(realDoc?.scribe ?? "{}")?.lastSummarySequenceNumber
if (lastSummarySequenceNumber === undefined) }
          Lumberjack.error(
					`Unable to delete ops. Reason: ....`,
					lumberjackProperties,
				);
				continue;
}
```

**Review by zhangxin511 — APPROVED** [review]

**Comment by arafat-java** [comment]
@nmsimons , @CraigMacomber , @tylerbutler 
This PR is approved but I am not authorized to merge this
Can someone please merge

**Pull request merged**